### ISSUE TITLE
wrong pin in history file

### ISSUE THREAD

**Issue by beckermr** [mamba-org/mamba#1024]
This sequence of installs produces an incorrect pin in the history file

```
conda create -n lsst-stack-test python==3.8 pip mamba
conda activate lsst-stack-test
mamba install stackvana=0
```

If you then inspect the history you see

```
...
+conda-forge/osx-64::wrapt-1.12.1-py38h5406a74_3
+conda-forge/osx-64::xorg-kbproto-1.0.7-h35c211d_1002
+conda-forge/osx-64::xorg-libx11-1.7.2-h0d85af4_0
+conda-forge/osx-64::xorg-libxau-1.0.9-h35c211d_0
+conda-forge/osx-64::xorg-libxdmcp-1.1.3-h35c211d_0
+conda-forge/osx-64::xorg-xproto-7.0.31-h35c211d_1007
+conda-forge/osx-64::xpa-2.1.20-h046ec9c_0
+conda-forge/osx-64::zfp-0.5.5-he49afe7_5
+defaults/osx-64::fftw-3.3.9-h9ed2024_1
# update specs: ['stackvana==0']
```

Notice that we did a compatible version install with `=` but mamba or mamba through conda wrote an exact version `==`.

**Comment by beckermr** [comment]
This happens even if mamba from the base env is used.

**Comment by beckermr** [comment]
The spec is also wrong if mamba is used to create the environment.

```
# update specs: ['python==3.8', 'pip', 'stackvana==0']
```

**Comment by beckermr** [comment]
cc @wolfv

**Comment by beckermr** [comment]
This command also results in the wrong pin (same incorrect one as above)

```
mamba create -n lsst-stack-test python==3.8 pip stackvana==0.*
```

**Comment by beckermr** [comment]
This bug is fixed in 0.14.1. Closing and thanks!
